easybuild/easyblocks/p/python.py

```diff
@@ -185,6 +198,11 @@ def configure_step(self):
         if self.cfg['optimized'] and LooseVersion(self.version) >= LooseVersion('3.5.4'):
             self.cfg.update('configopts', "--enable-optimizations")
 
+        # Pip is included since 3.4 via ensurepip https://docs.python.org/3.4/whatsnew/changelog.html
+        if LooseVersion(self.version) >= LooseVersion('3.4.0'):
+            # Default, but do it explicitly
+            self.cfg.update('configopts', "--with-ensurepip=upgrade")
```

Comment:
Instead of setting the default `upgrade`, I would just force the installation of `ensurepip` so that it is installed regardless of what is available in the system or the user distribution
```suggestion
            self.cfg.update('configopts', "--with-ensurepip=install")
```

Reply:
Iirc this is the same as the upgrade when using pip: I.e. Force installation even if already installed. I think install is less "pressing"

Reply:
You are right, I misread the docs. `--with-ensurepip=install` just does a regular install, so if any version of the package is already installed it won't do anything. `upgrade` at least will match the version shipped with Python.